package.json

```diff
@@ -36,6 +36,10 @@
   "jest": {
     "setupFilesAfterEnv": [
       "jest-extended/all"
+    ],
+    "verbose": true,
+    "testPathIgnorePatterns": [
+      "<rootDir>/__tests__/utils.test.js"
```

Comment:
Is there a reason you're ignoring these tests? If you just want to run the app tests then I'd take an opt-in approach and just run them by name (`jest nc-news`)